Org Protocol Handler.app/Contents/Resources/parse.py

```diff
@@ -1,25 +1,45 @@
 #!/usr/bin/env python
 
+import ConfigParser
+import os
+import subprocess
 import sys
-import urlparse
 import urllib
-import subprocess
-import os
-import ConfigParser
+import urlparse
 
 
-def get_emacsclient_path():
-    """Get the configured path to `emacsclient`, or the default."""
+def read_config():
     ini_path = os.path.expanduser("~/.orgprotocol.ini")
-    if os.path.exists(ini_path):
-        config = ConfigParser.ConfigParser()
-        try:
-            config.read([ini_path])
-            return config.get("emacsclient", "path")
-        except Exception, e:
-            pass
+    config = ConfigParser.ConfigParser()
+    try:
+        config.read([ini_path])
+    except Exception:
+        print("Error reading %s" % ini_path)
+    return config
+
+
+def emacs_client_command(config):
```

Comment:
Every function might as well have a documentation string. In this case I think it's important to point out that this returns a *list* as expected by `check_output` and not simply a string.

Apologies for my pedantry here, but if we're going to do this, we might as well do it right.

Reply:
It's no problem, I saw that they were over there so I just merged these. Thanks for all this work!